modules/custom_operations/user_ie_extensions/CMakeLists.txt

```diff
@@ -109,23 +109,29 @@ target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_OPENVINO_EXTENSION_A
 # TODO: remove
 target_include_directories(${TARGET_NAME} PUBLIC ./include/)
 
-# Wheel packaging using skbuild
-if(DEFINED SKBUILD)
-  # Installing the extension module to the root of the package
-  if(LINUX)
-    set_target_properties(${TARGET_NAME} PROPERTIES INSTALL_RPATH "$ORIGIN")
-    install(TARGETS ${TARGET_NAME} LIBRARY DESTINATION .)
-  elseif(APPLE)
-    set_target_properties(${TARGET_NAME} PROPERTIES INSTALL_RPATH "@loader_path")
-    install(TARGETS ${TARGET_NAME} LIBRARY DESTINATION .)
-  elseif(WIN32 AND X86_64)
-    install(TARGETS ${TARGET_NAME} RUNTIME DESTINATION .)
-  else()
-    message(FATAL_ERROR "Unsupported build platform")
-  endif()
+#
+# Installation rules
+#
 
-  if(extra_libs)
-    install(FILES ${extra_libs} DESTINATION .)
-  endif()
+include(GNUInstallDirs)
 
+# setting RPATH / LC_RPATH depending on platform
+if(LINUX)
+  set_target_properties(${TARGET_NAME} PROPERTIES INSTALL_RPATH "$ORIGIN")
+elseif(APPLE)
+  set_target_properties(${TARGET_NAME} PROPERTIES INSTALL_RPATH "@loader_path")
+endif()
+
+# Installing the extension module to the root of the package
+install(TARGETS ${TARGET_NAME}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT core
```

Comment:
When I did that change previously all dlls has been sabed in the bin subdir

Reply:
`bin` is a right location.

Reply:
Then we would need support this subdir in the init.py, now we a looking extension in the root install dir